.github/workflows/playwright.yml

```diff
@@ -0,0 +1,42 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [main, master]
```

Comment:
Why master is here?

```suggestion
    branches: [main]
```

Reply:
Same below

Reply:
Nope, it was autogenerated by playwright